FullStackMVCAPP/FullStackMVCAPP/Migrations/202108311613186_VirtualGOTContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿namespace FullStackMVCAPP.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class VirtualGOTContext : DbMigration
+    {
+        public override void Up()
+        {
+        }
+        
+        public override void Down()
+        {
+        }
+    }
+}
```

Comment:
was this migration necessary?

Reply:
I orginally thought if I changed the DbSet to a virtual one a migration is needed, I was wrong